MPF.Core/Data/Drive.cs

```diff
@@ -66,10 +68,21 @@ public class Drive
                 string? volumeLabel = Template.DiscNotDetected;
                 if (this.MarkedActive)
                 {
-                    if (string.IsNullOrEmpty(this.VolumeLabel))
-                        volumeLabel = "track";
-                    else
+                    if (!string.IsNullOrEmpty(this.VolumeLabel))
                         volumeLabel = this.VolumeLabel;
+                    else
+                    {
+                        RedumpSystem? system = this.GetRedumpSystem(null);
+                        if (system == RedumpSystem.SonyPlayStation || system == RedumpSystem.SonyPlayStation2)
```

Comment:
If you want to prepare for other possible systems in the future, it would be worthwhile changing this `if`/`else` into a `switch` to make it easier to expand.